eval.py

```diff
@@ -218,7 +218,7 @@
                 prob = torch.flip(prob, dims=[-1])
 
             # Probability mask -> index mask
-            out_mask = torch.argmax(prob, dim=0)
+            out_mask = torch.max(prob, dim=0).indices
```

Comment:
This was because of an PyTorch error on the MPS backend: https://github.com/pytorch/pytorch/issues/98191
Although it seems as though this issue has recently been resolved in the Nigtly build of PyTorch (but not the stable build 2.0.1) so should I amend this to use argmax as before?